css-values-4/Overview.bs

```diff
@@ -1063,33 +1063,69 @@ URL Processing Model</h4>
 
 		4. Let |base| be |sheet|'s <a spec=cssom>stylesheet base URL</a>. [[CSSOM]]
 
-		5. Let |referrer| be |documentBase|.
+		5. If |base| is null, set |base| to |documentBase|.
 
-		6. If |base| is null, set |base| to |documentBase|.
+		6. Let |url| be |urlValue|.
 
-		7. Let |parsedUrl| be the result of the [=URL parser=] steps with |url| and |base|.
-			If the algorithm returns an error, return.
+		7. If |urlValue| is a <<url>>, set |url| to |urlValue|'s [/=url].
 
-		8. If |corsMode| is "cors", set |referrer| to |sheet|'s
-			<a spec=cssom>location</a>. [[CSSOM]]
+		8. Let |parsedUrl| be the result of the [=URL parser=] steps with |url| and |base|.
+			If the algorithm returns an error, return.
 
 		9. Let |req| be a new [=/request=] whose [=request/url=] is |parsedUrl|, whose
 			[=request/destination=] is |destination|, [=request/mode=] is |corsMode|,
-			[=request/origin=] is |environmentSettings|'s [=environment settings object/origin=],
-			[=request/credentials mode=] is "same-origin", [=request/use-url-credentials flag=] is
-			set, [=request/client=] is |environmentSettings|, and whose [=request/referrer=] is
-			|referrer|.
+			[=request/origin=] is |environmentSettings|'s
+			[=environment settings object/origin=], [=request/credentials mode=] is
```

Comment:
Note that we do *not* wrap at a particular width, we wrap at semantic breaks in the text. The original wrapping here should be preserved.

Reply:
Fixed